lib/tiddle/token_issuer.rb

```diff
@@ -56,13 +56,13 @@ def authentication_token_class(resource)
       end
     end
 
-    def token_attributes(token_body, request, expires_in)
+    def token_attributes(token_body, request, expires_in, metadata: {})
```

Comment:
I think that it's time to turn this method to accept only keyword arguments, as we're adding the fourth argument.

Reply:
:+1: done!